### ISSUE TITLE
Standalone: VPinMAMEController::get_NVRAM Not implemented

### ISSUE THREAD

**Issue by francisdb** [vpinball/vpinball#1299]
**Describe the bug**
Tried playing `Space Station VR (Williams 1987) 1.1.0`
https://www.vpforums.org/index.php?app=downloads&showfile=14585

Got an error on these lines

```vbscript
	Dim NVRAM : NVRAM = Controller.NVRAM
	dim str : str = _
	ConvertBCD(NVRAM(CInt("&h200"))) & _
	ConvertBCD(NVRAM(CInt("&h201"))) & _
	ConvertBCD(NVRAM(CInt("&h202"))) & _
	ConvertBCD(NVRAM(CInt("&h203")))		'sys 11 current score
```

```
2023-12-29 12:21:19.613 WARN  [792770] [VPinMAMEController::get_NVRAM@555] Not implemented
2023-12-29 12:21:19.613 ERROR [792770] [CodeViewer::OnScriptError@1180] Script Error at line 205 : VBSE_ACTION_NOT_SUPPORTED
```

As an alternative standalone users can play `Space Station (Williams 1987) 1.21 `
https://www.vpforums.org/index.php?app=downloads&showfile=12717
However when playing this table in Desktop mode it will still fail. So the workaround is:

```vbs
'FloatingScores = Table1.ShowDT 
FloatingScores = 0
```

Which will disable floating scores that read nvram

**Comment by francisdb** [comment]
These functions have been implemented in 1f0605d89141cb6d1022499e1c400b3be2669247. Still to be tested.

**Comment by francisdb** [comment]
Confirming the fix, score popups working. Pretty cool.

![image](https://github.com/vpinball/vpinball/assets/161305/715e29ba-d471-44cb-a134-99e76b2606fd)
